### DISCUSSION TITLE
Define CONTRIBUTING guidelines and coding style guidelines.

### DISCUSSION THREAD

**Discussion by adityaruplaha** [Heroic-Games-Launcher/HeroicGamesLauncher#223] [Ideas]
As this repo continues to grow and attract more contributors, I think it will be helpful if coding guidelines, source control guidelines and style guidelines are provided to new wannabe contributors like myself. :)

This includes things like: trailing newlines, indent size, the formatter to use, rebase-over-merge, stuff like that.

---

Supersedes issue #45.

**Comment by Nocccer** [comment]
This is a great idea. Also if more people are involved in development, implementing tests (in my eyes) are necessary, too.

**Comment by flavioislima** [comment]
Nice.
Do you have any examples that we can use, some other open-source projects out there?

**Comment by adityaruplaha** [comment]
I'll have to see. Even if we don't, I might just write some myself :P.

**Comment by adityaruplaha** [comment]
UPDATE:
A basic contributing file is now in the wiki. I'm working on writing some coding guidelines.
